index.html

```diff
@@ -38,5 +38,106 @@ <h1>I'm Shimels, Glad to see you!</h1>
         <a href="#"> <img src="icones/Vector.png" alt="" /></a>
       </section>
     </main>
+
+    <main class="project-container">
+      <div class="card">
+        <img src="images/bk-image.jpg" alt="project image" />
+        <h2>Tonic</h2>
+        <h4>
+          <span>CANOPY</span> &nbsp; ● &nbsp; Back End Dev &nbsp; ● &nbsp; 2015
+        </h4>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod
+          euismod nisi, ac bibendum odio bibendum vel. Donec euismod nisl ut
+          magna aliquam, vel lacinia sapien bibendum. Sed eget velit vel velit
+          lacinia dapibus.
+        </p>
+        <ul class="tools">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+
+        <button class="see-project-btn">Show Project</button>
+      </div>
```

Comment:
- [ ] The button's texts overlap, kindly increase the width of the buttons to make the text on one line as in the Figma design to follow the project requirement.
### Your design :point_down: 
![Screenshot from 2023-07-07 17-00-39](https://github.com/shimels1/portfolio_microverse/assets/91142432/84004f5a-7e37-4e5a-b3be-a4b574c7781a)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-07-07 17-05-17](https://github.com/shimels1/portfolio_microverse/assets/91142432/61360eea-70b1-4df3-bc55-e4823c235421)